projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -50,9 +55,9 @@ const PieSemiCircleChart: FC< PieSemiCircleChartProps > = ( {
 	const { tooltipOpen, tooltipLeft, tooltipTop, tooltipData, hideTooltip, showTooltip } =
 		useTooltip< DataPointPercentage >();
 
-	const centerX = width / 2;
-	const height = width / 2;
-	const radius = width / 2;
+	const centerX = size / 2;
+	const height = size / 2;
```

Comment:
I think I missed this one - I don't quite get it why height is half the width and now half the size? @annacmc @retrofox

Reply:
I think this was an error on my behalf. size == height and size == width because of the 1:1 ratio nature of a circle. But I think the height = size / 2 is a mistake in there. I'll check it out and fix up in my next PR.